pkgs/racket-doc/scribblings/reference/pairs.scrbl

```diff
@@ -1255,6 +1255,28 @@ Returns a list with all elements from @racket[lst], randomly shuffled.
   (shuffle '(1 2 3 4 5 6))]}
 
 
+@defproc[(combinations [lst list?] [size (or/c #f exact-nonnegative-integer?) #f])
```

Comment:
Nitpick: I'd document it as two separate arities, as opposed to using an optional argument. The function really has two separate modes.